tests/unit_tests/test_unit.py

```diff
@@ -16,7 +16,7 @@
 from casanovo.data import ms_io
 from casanovo.data.datasets import SpectrumDataset, AnnotatedSpectrumDataset
 from casanovo.denovo.evaluate import aa_match_batch, aa_match_metrics
-from casanovo.denovo.model import Spec2Pep, _aa_pep_score
+from casanovo.denovo.model import Spec2Pep, _aa_pep_score, _calc_match_score
```

Comment:
**todo:** The integration tests are good in the other file, but we also need more detailed unit tests for the different functionalities individually.

Reply:
After making the tests for `_calc_match_score` more comprehensive, I don't see any more parts of the code that can be tested individually. The bulk of new additions are on the model side and require an integration test to run.